project-canvas/src/components/backlog-view/SprintsColumn.tsx

```diff
@@ -0,0 +1,106 @@
+import { Accordion, Badge, Flex, Group, Title, Text } from "@mantine/core"
+import { IconChevronRight } from "@tabler/icons"
+import { Issue, Sprint } from "project-extender"
+import { pluralize, storyPointsAccumulator } from "./backlogHelper"
+import { Column } from "./Column"
+
+export function SprintsColumn({
+  columns,
+  sprints,
+}: {
+  columns: Map<string, { id: string; list: Issue[] }>
+  sprints: Map<string, Sprint>
+}) {
+  return (
+    <Accordion
+      variant="separated"
+      radius="md"
+      chevron={<IconChevronRight />}
+      chevronPosition="left"
+      multiple
+      styles={(theme) => ({
+        chevron: {
+          "&[data-rotate]": {
+            transform: "rotate(90deg)",
+          },
+        },
+        control: {
+          padding: theme.spacing.xs,
+        },
+        item: {
+          border: "solid 1px lightgray",
+          "&:hover": {
+            backgroundColor: theme.colors.gray[1],
+          },
+        },
+      })}
+    >
+      {Array.from(columns.keys())
+        .filter((columnName) => columnName !== "Backlog")
+        .sort((a, b) => {
+          if (
+            sprints.get(a)!.state === "active" &&
+            sprints.get(b)!.state !== "active"
+          ) {
+            return -1
+          }
+          if (
+            sprints.get(a)!.state !== "active" &&
+            sprints.get(b)!.state === "active"
+          ) {
+            return 1
+          }
+          return a.localeCompare(b)
```

Comment:
Can we refactor this to a function called sortByActive()? That way it's clear what this does.

Reply:
This can be in the same file just declared above

Reply:
done